From aff1984acb0809c69cffbb2c1c29334256c69cba Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Wed, 24 Apr 2019 11:35:00 +0530
Subject: [PATCH] Fix build error due to api prototype change

Build issue seen due to wifi_start_sending_offloaded_packet
prototype change.

Fix the error by using latest prototype.

Tracked-On: OAM-84141
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 lib/wifi_hal.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/wifi_hal.cpp b/lib/wifi_hal.cpp
index d62f6ba..726841b 100644
--- a/lib/wifi_hal.cpp
+++ b/lib/wifi_hal.cpp
@@ -545,8 +545,9 @@ wifi_error wifi_set_bssid_hotlist(wifi_request_id id, wifi_interface_handle ifac
 }
 
 wifi_error wifi_start_sending_offloaded_packet(wifi_request_id id,
-		wifi_interface_handle iface, u8 *ip_packet, u16 ip_packet_len,
-		u8 *src_mac_addr, u8 *dst_mac_addr, u32 period_msec) {
+		wifi_interface_handle iface, u16 ether_type, u8 *ip_packet,
+		u16 ip_packet_len, u8 *src_mac_addr, u8 *dst_mac_addr,
+		u32 period_msec) {
 	return WIFI_ERROR_NOT_SUPPORTED;
 }
 
-- 
2.17.1

